frontend/ui/elements/screen_fullscreen_menu_table.lua

```diff
@@ -0,0 +1,36 @@
+local _ = require("gettext")
+local android = require("device/android/device")
```

Comment:
Perhaps `local _, android = pcall(require, "android")` and skip the Device stuff up there?